src/test/java/org/littleshoot/proxy/HttpFilterTest.java

```diff
@@ -58,8 +73,14 @@ public void tearDown() throws Exception {
                 webServer.stop();
             }
         } finally {
-            if (proxyServer != null) {
-                proxyServer.stop();
+            try {
+                if (proxyServer != null) {
+                    proxyServer.stop();
```

Comment:
Good idea. We need to go through and do this to all the other unit tests as well.